src/modules/styles/colors.ts

```diff
@@ -0,0 +1,3 @@
+export const PRIMARY_NAVY_BLUE = '#000080';
```

Comment:
Declare it as a object. We can just import the whole object and take the necessary color from it. Something like
`
export const colors = {
PRIMARY_NAVY_BLUE = '#000080';
}
`